src/main/java/io/jenkins/plugins/orka/OrkaAgent.java

```diff
@@ -143,6 +154,64 @@ public String getJvmOptions() {
         return this.jvmOptions;
     }
 
+    public List<PortMapping> getPortMappins() {
+        return portMappings;
+    }
+
+    public void processPortMappings() {
```

Comment:
Ping about this comment: https://github.com/jenkinsci/macstadium-orka-plugin/pull/137#discussion_r1843751117